app/controllers/organizations_controller.rb

```diff
@@ -143,4 +156,17 @@ def update_organization_params
       .require(:organization)
       .permit(:title)
   end
+
+  def verify_organization_user
+    @removed_user = User.find(params[:user_id])
+    not_found unless @removed_user && current_organization.users.map(&:id).include?(@removed_user.id)
+  end
+
+  def transfer_assignments
+    new_owner = current_organization.users.where.not(id: @removed_user.id).first
+    current_organization.all_assignments.map do |a|
+      a.creator_id = new_owner.id if a.creator_id == @removed_user.id
+      a.save
```

Comment:
You could clean this up a bit with `next`

```ruby
next unless assignment.creator_id == @removed_user.id
assignment.update_attributes(creator_id: new_owner_id)
```

Reply:
Awesome ! I'm not used to `next` but it seems great (gonna keep this for the next time it happens)